client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -349,16 +359,50 @@ class DomainManagementNavigationEnhanced extends React.Component {
 	}
 
 	getSecurity() {
-		const { selectedSite, translate } = this.props;
+		const { selectedSite, domain, translate } = this.props;
+
+		const shouldRenderDomainSecurity = config.isEnabled(
+			'domains/new-status-design/security-option'
+		);
+
+		if ( ! shouldRenderDomainSecurity ) {
+			return null;
+		}
+
+		const { pointsToWpcom, sslStatus } = domain;
+
+		if ( ! pointsToWpcom || ! sslStatus ) {
+			return null;
+		}
+
+		let sslStatusHuman;
+		switch ( sslStatus ) {
+			case sslStatuses.SSL_PENDING:
+				sslStatusHuman = translate( 'provisioning', {
+					comment: 'Shows as "HTTPS encryption: provisioning" in the nav menu',
+				} );
+				break;
+			case sslStatuses.SSL_ACTIVE:
+				sslStatusHuman = translate( 'on', {
+					comment: 'Shows as "HTTPS encryption: on" in the nav menu',
+				} );
+				break;
+			case sslStatuses.SSL_DISABLED:
+				sslStatusHuman = translate( 'disabled', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **36** times:
`translate( 'Disabled.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/new-domain-security-screen)